### DISCUSSION TITLE
blog/minimalisme-entrepreneurial/

### DISCUSSION THREAD

**Discussion by giscus** [svandegar/seraphin#7] [Announcements]
# blog/minimalisme-entrepreneurial/

Savil Lavingia à fondé Gumroad et levé plusieurs millions de dollars avant de faire marche arrière et revenir à un modèle plus sain : le minimalisme entrepreneurial.
Dans cet article, je partage avec vous ce que j’ai retenu de la lecture de son livre “The Minimalist Entrepreneur”. Comment développer un projet sans tomber dans le piège de l’hyper-croissance.

https://seraphin.dev/blog/minimalisme-entrepreneurial/

**Comment by yumaollivier** [comment]
Voilà un article clair et concis comme je les aime !
J'avais encore quelques petites parts d'ombre sur la façon de voir les choses et me voilà plus enclin à démarrer une nouvelle aventure 🌱
Merci beaucoup pour ce partage 👌

**Reply by svandegar** [reply]
Merci beaucoup pour ton commentaire, Yuma! 

Content de savoir que cet article est utile :-)
